backend/app/controllers/charities_controller.rb

```diff
@@ -4,37 +4,35 @@ class CharitiesController < ApplicationController
   before_action :authenticate_admin!, only: %i[create update destroy]
   before_action :set_charity, only: %i[show update destroy]
 
-  wrap_parameters format: :json, include: %w[name description websiteUrl logoBase64 imageBase64]
-
   def index
     @charities = Charity.all
   end
 
   def show; end
 
   def create
-    @charity = Charity.new(charity_params.except(:logo_base64, :image_base64))
-    @charity.logo.attach(data: charity_params[:logo_base64]) unless charity_params[:logo_base64].nil?
-    @charity.image.attach(data: charity_params[:image_base64]) unless charity_params[:image_base64].nil?
+    @charity = Charity.new(charity_params)
+    @charity.logo.attach(data: params[:logo_base64]) unless image_params[:logo_base64].nil?
+    @charity.image.attach(data: params[:image_base64]) unless image_params[:image_base64].nil?
     @charity.save!
```

Comment:
Same here. Let's use `image_params` instead of params.

Also help to flip the logic as per CampaignsController.